app/styles/helpers/inverse-element.less

```diff
@@ -23,13 +41,17 @@
   .upf-select,
   .selectize-input {
     color: @color-text-contrast;
-    background-color: @upf-primary-rock-blue;
-    border-color: @upf-primary-rock-blue;
+    background-color: fade(white, 10%);
```

Comment:
I would rather use `fadeout(color, 90%)`  than `fade(color, 10%)`. It's just a preference as I think `fadeout` tells more about what it does: Setting Opacity. And also looks more like than CSS's `opacity` rule